Jenkinsfile.oscontainer

```diff
@@ -22,12 +22,17 @@ node(NODE) {
     }
 
     withCredentials([
-        string(credentialsId: params.REGISTRY, variable: 'REGISTRY'),
+        usernameColonPassword(credentialsId: params.REGISTRY_CREDENTIALS, variable: 'CREDS'),
     ]) {
-        docker.withRegistry("${REGISTRY}", params.REGISTRY_CREDENTIALS) {
+        def (username, password) = "${CREDS}".split(':')
+        sh """set +x
+              echo docker login -e 'unused@example.com' -u 'username' -p password registry.svc.ci.openshift.org
+              docker login -e 'unused@example.com' -u "${username}" -p "${password}" registry.svc.ci.openshift.org
```

Comment:
Just verifying, does the version of docker in use still support ``-e``? I believe it was dropped.

Reply:
Sadly because we don't apply updates to our CI nodes, it's actually required there right now.  It's a bit tempting to rebase them on CL or something, but let's not do that right now...